cmd/noms/noms_show.go

```diff
@@ -69,6 +74,13 @@ func runShow(args []string) int {
 		return 0
 	}
 
+	tz, err := locationFromTimezoneArg(tzName, nil)
```

Comment:
I did make the messages consistent but I'm not sure if it's worth sharing since they would still have to check the err and return.